xml/System.Diagnostics.Metrics/ObservableGauge`1.xml

```diff
@@ -57,7 +57,7 @@ This class supports only the following generic parameter types: <xref:System.Byt
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Observe() fetches the current measurements being tracked by this observable counter.</summary>
+        <summary>Fetches the current measurements being tracked by this observable counter.</summary>
         <returns>To be added.</returns>
```

Comment:
Any objections to adding the missing returns value, since we're already here?:

```suggestion
        <returns>An enumeration of measurements of type `<typeparamref name="T"/>` that are being tracked by this observable counter.</returns>
```

Reply:
Since `T` isn't listed as a `typeparam`, presumably because it's not specified by the caller, do you think it's okay to use `typeparamref`?

Reply:
I guess because this is a sealed class, these custom comments don't show up anyway. Only inherited members are shown.

https://review.docs.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.observablegauge-1?view=net-6.0&branch=pr-en-us-7510